src/solver/infeasible-problem-analysis/report.cpp

```diff
@@ -86,6 +84,10 @@ void InfeasibleProblemReport::logSuspiciousConstraints()
     {
         Antares::logs.error() << c.prettyPrint();
     }
+}
+
+void InfeasibleProblemReport::logInfeasibilityCauses()
```

Comment:
Here we split logs into 2 separate functions : 
- one that logs suspicious constraints 
- one that logs of infeasibility causes